### PR TITLE
edit column address in table order_details and fix seed,factory,show …

### PR DESCRIPTION
**Pull request by ghost**
…order detail

## Description
delete column address in table order_details, add column address into table orders, fix seed, fix factory and fix show order details
## Reference Requirements
- Task ID: TI-1

## Migrations
YES

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-sonle** [comment]
LGTM

File: database/migrations/2018_07_01_082100_remove_order_details_column.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('order_details', function($table) {
+            $table->integer('address');
```
Comment (at-tuanphan) [inline database/migrations/2018_07_01_082100_remove_order_details_column.php]: `address` sao lại type là `integer` em?

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — APPROVED** [review]
LGTM [:+1:](http://lgtm.in/u/0aNBSIYwn)
[LGTM](http://lgtm.in/i/0aNBSIYwn)

**Pull request merged**